server/src/infra/repositories/asset.repository.ts

```diff
@@ -454,8 +481,9 @@ export class AssetRepository implements IAssetRepository {
   getByTimeBucket(timeBucket: string, options: TimeBucketOptions): Promise<AssetEntity[]> {
     const truncateValue = truncateMap[options.size];
     return this.getBuilder(options)
-      .andWhere(`date_trunc('${truncateValue}', "fileCreatedAt") = :timeBucket`, { timeBucket })
-      .orderBy('asset.fileCreatedAt', 'DESC')
+      .andWhere(`date_trunc('${truncateValue}', "localDateTime") = :timeBucket`, { timeBucket })
+      .orderBy(`date_trunc('day', "localDateTime")`, 'DESC')
+      .addOrderBy('asset.localDateTime', 'DESC')
```

Comment:
Why do we need both of these? It seems like one would be sufficient. Maybe would would need both if you were sorting by `fileCreatedAt` somewhere.

Reply:
Ah, I am sorting the second one on the wrong field. Will correct to fileCreatedAt. I think.